eng/dict/rust-custom.txt

```diff
@@ -3,3 +3,4 @@ newtype
 repr
 rustc
 rustls
+turbofish
```

Comment:
Is this established enough? It is listed [here](https://doc.rust-lang.org/book/appendix-02-operators.html?highlight=turbofish#:~:text=generic%20type%20parameters.-,Table%20B%2D4%3A%20Generics,-Symbol) so maybe.

Reply:
It's certainly official enough (IMO) to be in our cspell dict, so we can use it freely in internal comments. I'm actually using it in a doc comment, which also explains the term:

```rust
    /// We recommend using "turbofish" syntax (`query_items::<SomeTargetType>(...)`) to specify the target type, as it makes type inference easier.
```

I think that's a very reasonable doc comment and it's entirely appropriate to reference a common community term for the syntax while also explaining it. In fact, I'll put a link to your reference in there as an "official" citation ;).